mono/mini/method-to-ir.c

```diff
@@ -2176,7 +2175,7 @@ test_tailcall (MonoCompile *cfg, MonoBoolean tailcall)
 	// Do not change "tailcalllog" here without changing other places, e.g. tests that search for it.
 	//
 	g_assertf (tailcall || !mini_get_debug_options ()->test_tailcall_require, "tailcalllog fail from %s", cfg->method->name);
-	if (cfg->verbose_level)
+	if (cfg->verbose_level > 1 || mini_get_debug_options ()->tailcall)
```

Comment:
We should use `mono_trace` instead, it's exactly done for these kind of use cases.

Reply:
I see printf and g_print a lot and was unaware. Let me look more then.

Reply:
MONO_LOG_LEVEL=debug MONO_LOG_MASK=tailcall and I fill in flag_mask_map? I'll try that.
ok to also support -v -v, or we should stop adding anything under verbose_level?

Reply:
oh, no, can't really have -v -v also, that'd be messy. just mono_trace.

Reply:
notice method-to-ir.c never uses mono_trace..thus my mistake